packages/modules/viewer-react/src/services/iModel/iModelViewerHelper.ts

```diff
@@ -0,0 +1,102 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
nit: for consistency, could you export this file in the barrel in this dir? and then consume exports from here from that barrel in the other files?

Reply:
done